project-base/app/src/Model/Product/Flag/FlagRepository.php

```diff
@@ -73,10 +72,9 @@ public function getVisibleFlagById(int $flagId, string $locale): Flag
      */
     public function getAllVisibleFlags(string $locale): array
```

Comment:
first, I wanted to ask you to move the methods to the framework package, but it would make sense to move it along with the API functionality. It is probably not as easy as it seems so I¨ll leave the decision up to you, I do not insist on that :smile: